### ISSUE TITLE
The NLnet Labs testbed TAL was updated

### ISSUE THREAD

**Issue by AlexanderBand** [NLnetLabs/routinator#676]
installing it on 0.10.2 using `--tal nlnetlabs-testbed` results in

```
Jan 13 20:58:23 debian11-2vcpu-2gb-ams3-01 routinator[2566]: No valid trust anchor for TAL nlnetlabs-testbed
```

**Comment by AlexanderBand** [comment]
I see this is addressed in PR #637.

**Comment by partim** [comment]
ITYM: Fixed in #637.
